File: CS50/pset6/mario/mario.py
from cs50 import get_int

# Asks height from user
while True:
    h = get_int("Height: ")
    if h in range(24):
        break
# Builds pyramid
for k in range(h):
    # add \n (h - k + 2) times
    print(" " * (h - k - 1), end='')
    print("#" * (k + 2), end='')
    print("")

File: CS50/pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)

{
    // Asks height from user
    int k;
    int h;
    do
    {
        h = get_int("Height: ");
    }
    while (h < 0 | h > 23);

    // Builds pyramid
    for (k = 0; k < h; k++)
    {
        //add \n h-k+2 times
        for (int i = 0; i < (h - k - 1); i++)
        {
            printf(" ");
        }
        for (int j = 0; j < (k + 2); j++)
        {
            printf("#");
        }

        printf("\n");
    }
}
